src/main/java/com/internet/shop/util/HashUtil.java

```diff
@@ -0,0 +1,38 @@
+package com.internet.shop.util;
+
+import com.internet.shop.model.User;
+import java.security.MessageDigest;
+import java.security.NoSuchAlgorithmException;
+import java.security.SecureRandom;
+import org.apache.log4j.Logger;
+
+public class HashUtil {
+    private static final String ALGORITHM = "SHA-512";
+    private static Logger logger = Logger.getLogger(HashUtil.class);
```

Comment:
Why do you need logger?

Reply:
I thought I'd need to log the exceptions. like in the video, but then read the CheckList and removed it, but forgot to remove the Logger and dependency) Thanks))